2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2020 @@
+scenario, but transferring software use to new problems, domains, and/or
+applications, justifies developing research software with a long-term
+perspective as *sustainable research software*.
+
+In order to support research, a sustainable software must be
+correct (Hatton 2007; Chang et al. 2006; Matthews 2007), validatable,
+understandable, documented, publicly released, adequately published
+(i.e. in persistently identifiable form as software source code (Smith
+et al. 2016), and potentially in an additional paper which describes the
+software concept, design decisions and development rationale), actively
+maintained, and  (Merali 2010; Barnes 2010; Tse 2010).
+
+To make it possible to reproduce computationally supported research, a
+software that has been used must continue to exist in the first place,
+but also continue to be usable, understandable, and reproducible in the
+evolving software and hardware environment. Moreover, the software
+should support reuse scenarios to avoid duplication of efforts, and
+unneeded drain of resources. If it is publicly funded, it should also be
+open source and freely accessible.
+
+Currently, creating and using sustainable research software is not yet
+sufficiently incentivized. To evaluate where to address this
+shortcoming, we have identified the following challenges:
+
+-   **Lack of benefit for the individual**: The primary motivation for
+    sustainable research software is the common benefit, rather than the
+    individual benefit. While directing resources towards sustainable
+    research software may in the short run yield disadvantages for the
+    developing individual (e.g., fewer paper publications) (Bangerth and
+    Heister 2014; Prins et al. 2015), it is clearly beneficial for the
+    research community as a whole, as it enables better and more
+    research by freeing funds for actual research rather than
+    (repetitive) software development.
+
+-   **Lack of awareness**: Research software sustainability
+    (cf. (Venters et al. 2014; Goble 2014; Druskat 2016; Katz 2018)) and
+    its importance is lacking acceptance, and research software
+    engineering in its implementation as sustainable software
+    development and software maintenance is not sufficiently supported,
+    both in Germany and beyond (Brett et al. 2017; Akhmerov, Cruz,
+    Drost, Hof, Knapen, Kuzak, Martinez-Ortiz, et al. 2019; Casties et
+    al. 2019).
+
+-   **Lack of expertise**: Knowledge about how to create, maintain, and
+    support sustainable research software is emerging (Wilson et al.
+    2014, 2017; Stodden and Miguez 2014) but has not yet permeated
+    related activities within organizations, specifically teaching,
+    mentoring, and consultancy. This lack of expertise can also lead to
+    divergence between software design and community uptake, e.g., if
+    the software fails to meet the needs of the target group, or is
+    insufficiently usable. Research Software Engineers combine
```

Comment:
I don't quite understand what you want to say here. Maybe rephrase?

Reply:
1. Lack of training for such expertise
2. Something with "lack of knowledge transfer and research community uptake", and usability
3. Something RSEs are ...

Probably the whole paragraph wants to mention only issue 1